src/core/webidl-clipboard.js

```diff
@@ -30,12 +30,15 @@ async function createButton() {
 export async function run() {
   // This button serves a prototype that we clone as needed.
   const copyButton = await copyButtonPromise;
-  for (const pre of document.querySelectorAll("pre.idl")) {
+  for (const idlHeader of document.querySelectorAll("div.idlHeader")) {
     const button = copyButton.cloneNode(true);
+    const { textContent: idlText } = idlHeader
```

Comment:
Can we use `idlHeader.nextElementSibling`?
Might be better to iterate over `pre.idl` and then find `idlHeader` and `span` though

Reply:
Just realized there could be multiple `<span>`s 

```
<pre>
dict Foo {};
interface Y {};
</pre>
```